### DISCUSSION TITLE
Hi everyone

### DISCUSSION THREAD

**Discussion by Isotrom** [community/community#130563] [GitHub Education]
### Hi everyone,

I have received such a message "Congrats! Your request for [@Isotrom] was approved and your benefits will be available [soon]." However, I still can't use the copilot. Is it a normal situation? Thanks!

**Comment by davevad93** [comment]
Hi @Isotrom ,

Yes, it's normal, you still have to wait a few days for your benefits to be available, when they are available the status will change to:

`Congrats! Your request for @Isotrom  was approved and your benefits are now available`

**Reply by Isotrom** [reply]
Hi @davevad93,
Thanks a lot for your reply! Have a good day!

**Reply by davevad93** [reply]
You're welcome @Isotrom , have a nice day too!
